test/integration/api/external/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML+ARIA 1.0//EN" "http://www.w3.org/WAI/ARIA/schemata/html4-aria-1.dtd">
```

Comment:
enlighten me as to why such verbosity in doctype is essential for these tests?

Reply:
This was copied from other axe-core test harnesses. It's not essential to this test.